src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
+});
+
+export default store;
```

Comment:
- [ ] According to the project requirements, you are expected to import the created reducers for books and categories in this file and pass them inside the reducer object.

- [ ] You could set up your reducer like the code below: 👍🏽
- [ ] Note: The bookReducer and categoriesReducer should be imported at the top of the file before using them 👍🏽

```jsx
const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})
```